[2022-06-27 13:31:07,381] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-06-27 13:31:07,417] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-06-27 13:31:07,418] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:07,419] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:31:07,420] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:07,456] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_B> on 2022-02-07 00:00:00+00:00
[2022-06-27 13:31:07,471] {standard_task_runner.py:52} INFO - Started process 57027 to run task
[2022-06-27 13:31:07,492] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_B', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp23wpj06d', '--error-file', '/tmp/tmp0z7zkby8']
[2022-06-27 13:31:07,524] {standard_task_runner.py:80} INFO - Job 192: Subtask model_B
[2022-06-27 13:31:07,732] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_B scheduled__2022-02-07T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:31:07,869] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-06-27 13:31:07,872] {python.py:173} INFO - Done. Returned value was: 6
[2022-06-27 13:31:07,970] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_B, execution_date=20220207T000000, start_date=20220627T133107, end_date=20220627T133107
[2022-06-27 13:31:08,072] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:31:08,211] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
